src/provider.ts

```diff
@@ -24,6 +24,12 @@ export interface RequestInvoiceArgs {
   defaultMemo?: string;
 }
 
+export interface KeysendArgs {
+  destination: string;
+  customRecords: Record<string, string>;
```

Comment:
Seems annoying to make this a required parameter, is there any reason why it wouldn't be
```suggestion
  customRecords?: Record<string, string>;
```
and up to to the implementation to handle when the argument is undefined? While the `webln` library is typed, there are sure to be a lot of devs who just use JS and won't realize that they _have_ to pass this argument, so you'll likely want to make your implementation resistant to a missing `customRecords` key.

Reply:
+1 for having this optional. customRecords are not necessary and keysend payments can be done without them, too.